tbls/v2/tblsconv/tblsconv.go

```diff
@@ -62,6 +62,17 @@ func PubkeyFromBytes(data []byte) (v2.PublicKey, error) {
 	return *(*v2.PublicKey)(data), nil
 }
 
+// PubkeyFromCore returns a v2.PublicKey from the given core public key.
+// Returns an error if the data isn't of the expected length.
+func PubkeyFromCore(pk core.PubKey) (v2.PublicKey, error) {
```

Comment:
nit: a test case for this would be nice, but considering both `Bytes()` and `PubkeyFromBytes()` are tested, it's not 100% necessary.

Reply:
wll do